src/drivers/amd/rembrandt/ipc.c

```diff
@@ -31,66 +31,21 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <platform/platform.h>
-
-/* 49be8ff3-71a3-4456-bb7e-4723f2e5730c */
-DECLARE_SOF_UUID("rembrandt-ipc-task", ipc_task_uuid, 0x49be8ff3, 0x71a3, 0x4456,
-		 0xbb, 0x7e, 0x47, 0x23, 0xf2, 0xe5, 0x73, 0x0c);
+#include <platform/ipc.h>
 
 volatile acp_scratch_mem_config_t *pscratch_mem_cfg = (volatile acp_scratch_mem_config_t *)
 						      (PU_SCRATCH_REG_BASE + SCRATCH_REG_OFFSET);
-static inline uint32_t sof_ipc_host_status(void)
-{
-	return (pscratch_mem_cfg->acp_host_ack_write | pscratch_mem_cfg->acp_host_msg_write);
-}
-
-static  inline uint32_t sof_ipc_host_msg_flag(void)
-{
-	return pscratch_mem_cfg->acp_host_msg_write;
-}
-
-static  inline uint32_t sof_ipc_host_ack_flag(void)
-{
-	return pscratch_mem_cfg->acp_host_ack_write;
-}
-
-static inline uint32_t sof_ipc_dsp_status(void)
-{
-	return (pscratch_mem_cfg->acp_dsp_msg_write | pscratch_mem_cfg->acp_dsp_ack_write);
-}
-
-static inline void sof_ipc_host_ack_clear(void)
-{
-	pscratch_mem_cfg->acp_host_ack_write = 0;
-}
 
-static inline void sof_ipc_host_msg_clear(void)
-{
-	pscratch_mem_cfg->acp_host_msg_write = 0;
-}
-
-static inline void sof_ipc_dsp_ack_set(void)
-{
-	pscratch_mem_cfg->acp_dsp_ack_write = 1;
-}
-
-static inline void sof_ipc_dsp_msg_set(void)
-{
-	pscratch_mem_cfg->acp_dsp_msg_write = 1;
-}
-
-static void irq_handler(void *arg)
+void irq_handler(void *arg)
```

Comment:
> if you export this function, please make its name more specific

Since this function is declared and used in IPC specific functions. we feel this name is ok as we can understand that this is used for IPC functionality.

Reply:
The name is quite generic and may collide with IRQ core logic and result in build errors. This can be fixed by calling it ```rmb_irq_handler```

Reply:
> The name is quite generic and may collide with IRQ core logic and result in build errors. This can be fixed by calling it `rmb_irq_handler`

Currently, we don't see any build issues, we will do this change in the next PRs. Currently, we need Vangogh to be merged in the main.

Reply:
You cannot grab a name as generic as `irq_handler` in product-specific code because it instantly blocks everyone from using in common code.

We all have deadlines and we all know that "future PRs" can take a very long time because cleanups always have lower priority compared to other deadlines.

Reply:
Had updated the `irq_handler` function name.